src/bugsnag.cr

```diff
@@ -35,12 +35,11 @@ module Bugsnag
           headers = HTTP::Headers.new
           headers["Content-Type"] = "application/json"
           headers["Bugsnag-Api-Key"] = config.api_key
-          headers["Bugsnag-Payload-Version"] = 5
+          headers["Bugsnag-Payload-Version"] = "5"
           headers["Bugsnag-Sent-At"] = Time.utc.to_rfc3339
           url = (config.use_https ? "https:" : "http:") + "//notify.bugsnag.com"
 
           HTTP::Client.post(url, headers, report.to_json)
-          puts "Report sent to bugsnag!".colorize(:red)
```

Comment:
since this is in a `spawn` you'll never see it from your main log. We should probably also figure out how to handle the response...